opsdroid/web.py

```diff
@@ -25,8 +25,10 @@ def __init__(self, opsdroid):
         self.web_app = web.Application()
         self.runner = web.AppRunner(self.web_app)
         self.site = None
-        self.web_app.router.add_get("/", self.web_index_handler)
-        self.web_app.router.add_get("", self.web_index_handler)
+        if not self.config.get('disable_web_index_handler_in_root', False):
```

Comment:
Self-reviewing my code here now. As YAML does inherently treat booleas strings, should we then compare the value against string "true"  instead of a simple not check?